lib/nylas/resources/redirect_uris.rb

```diff
@@ -13,9 +13,9 @@ class RedirectUris < Resource
 
     # Return all redirect uris.
     #
-    # @return [Array(Array(Hash), String)] The list of redirect uris and API Request ID.
+    # @return [Array(Array(Hash), String, String)] The list of redirect uris, API Request ID, and next cursor.
     def list
-      get(
+      get_list(
```

Comment:
Same comment as the grants endpoint - I believe our UAS endpoints uses limt/offset pagination